public/static/locales/en/apps.json

```diff
@@ -84,33 +84,42 @@
     "publicLabel": "Public",
-    "publicSubmissionTitle": "Publish {{appName}}",
-    "publicSubmissionPrompt": "After saving your app and testing it, you may use this form to submit it for public use so it can be used by others. This form contains the required information needed for our team to publish your app. Please contact <support>support</support> to submit your test data for public use. <br/> <b>Note</b>: If you are submitting a workflow/pipeline, every app used in that workflow must be public before the workflow can be published.",
     "publicQLTooltip": "Quick Launch visible to all users",
-    "publishedOn": "Published on:",
-    "publicationRequestSuccess": "Your request to publish {{appName}} has been submitted. You will be notified when the request is complete.",
+    "publicSubmissionPrompt": "After saving your app and testing it, you may use this form to submit it for public use so it can be used by others. This form contains the required information needed for our team to publish your app. Please contact <support>support</support> to submit your test data for public use. <br/> <b>Note</b>: If you are submitting a workflow/pipeline, every app used in that workflow must be public before the workflow can be published.",
+    "publicSubmissionTitle": "Publish {{appName}}",
     "publicationRequestFailed": "Unable to submit your request. Please try again.",
+    "publicationRequestSuccess": "Your request to publish {{appName}} has been submitted. You will be notified when the request is complete.",
+    "publish": "Publish",
+    "publishApp": "Publish {{appName}}",
+    "publishedOn": "Published on:",
     "qLaunchEmbedToolTip": "Copy embed code",
     "qLaunchShareToolTip": "Share Quick Launch URL",
     "qLaunchToolTip": "Use this Quick Launch",
     "quickLaunch": "Quick Launch",
-    "quickLaunchError": "Unable to get Quick Launch.",
+    "quickLaunchCreateSuccess": "Quick Launch {{name}} created successfully.",
     "quickLaunchDeleteConfirmation": "Do you wish to delete this Quick Launch? Note: Deleting a public Quick Launch will break embedded and shared links.",
+    "quickLaunchDeleteError": "Unable to delete {{name}}.",
+    "quickLaunchError": "Unable to get Quick Launch.",
     "quickLaunchLabel": "Quick Launch",
     "quickLaunchNameLabel": "Name",
     "quickLaunchNotSupportedMessage": "Quick Launch not supported for Agave apps!",
-    "quickLaunchCreateSuccess": "Quick Launch {{name}} created successfully.",
-    "quickLaunchDeleteError": "Unable to delete {{name}}.",
     "rateApp": "Rate {{appName}}",
     "rating": "Rating",
     "ratingMutationError": "Unable to update rating. Please try again.",
     "references": "References",
     "removeFromFavorites": "Remove From Favorites",
     "save": "Save",
+    "selectApps": "Select apps",
     "selectedCategoryAriaLabel": "Selected app category",
     "selectedCategoryAriaMenuItemLabel": "Selected app category menu items",
+    "selectedItems": "{{count}} selected",
```

Comment:
My IDE sorted this file - only `selectedItems` and `selectApps` should have been added